server/models/helpers/SearchHelper.test.ts

```diff
@@ -207,48 +206,119 @@ describe("SearchHelper", () => {
         createdById: user.id,
         title: "test",
       });
+      await buildDocument({
+        userId: user.id,
+        teamId: user.teamId,
+        createdById: user.id,
+        title: "test",
+      });
+      await buildDocument({
+        userId: user.id,
+        teamId: user.teamId,
+        createdById: user.id,
+        title: "test",
+        archivedAt: new Date(),
+      });
       const { results } = await SearchHelper.searchForUser(user, "test", {
-        includeDrafts: true,
+        statusFilter: [StatusFilter.Draft],
       });
       expect(results.length).toBe(1);
     });
 
-    test("should not include drafts", async () => {
+    test("should search only published created by user", async () => {
       const user = await buildUser();
+      await buildDocument({
+        title: "test",
+      });
       await buildDraftDocument({
         teamId: user.teamId,
         userId: user.id,
         createdById: user.id,
         title: "test",
       });
+      await buildDocument({
+        userId: user.id,
+        teamId: user.teamId,
+        createdById: user.id,
+        title: "test",
+      });
+      await buildDocument({
+        userId: user.id,
+        teamId: user.teamId,
+        createdById: user.id,
+        title: "test",
+        archivedAt: new Date(),
+      });
       const { results } = await SearchHelper.searchForUser(user, "test", {
-        includeDrafts: false,
+        statusFilter: [StatusFilter.Published],
       });
-      expect(results.length).toBe(0);
+      expect(results.length).toBe(1);
     });
 
-    test("should not include drafts with user permission", async () => {
```

Comment:
@tommoor I'm not sure what this test does and if it still applies. Can you help me with this? Do we still want to keep this condition?

Reply:
The test is checking that `includeDrafts=false` filters correctly when there is a user membership to the document. Some equivalent should be easy enough to keep

Reply:
Yup. I've re-added the tests with some minor changes. Can you verify this once when you have time?